trusted-firmware-m/platform/ext/target/nordic_nrf/nrf5340dk_nrf5340_cpuapp/RTE_Device.h

```diff
@@ -40,13 +40,13 @@
 #define   RTE_USART1                    1
 //   <h> Pin Selection (0xFFFFFFFF means Disconnected)
 //     <o> TXD
-#define   RTE_USART1_TXD_PIN            25
+#define   RTE_USART1_TXD_PIN            33
 //     <o> RXD
-#define   RTE_USART1_RXD_PIN            26
+#define   RTE_USART1_RXD_PIN            32
 //     <o> RTS
-#define   RTE_USART1_RTS_PIN            0xFFFFFFFF
+#define   RTE_USART1_RTS_PIN            11
```

Comment:
The secure UART in TF-M does not use flow control, so we don't need to assign any pins as RTS and CTS lines. Actually, even RXD is not needed, as the console is output-only.